### PR TITLE
Allow to configure the Bundler version to use

### PR DESCRIPTION
**Pull request by sschuberth**
Please have a look at the individual commit messages for the details.

### CODE REVIEW

File: analyzer/src/main/kotlin/managers/Bundler.kt [hunk 1]
```diff
@@ -78,6 +76,11 @@ private const val RESOLVE_DEPENDENCIES_SCRIPT = "scripts/bundler_resolve_depende
  */
 private const val BUNDLER_GEM_NAME = "bundler"
 
+/**
```
Comment (fviernau) [inline analyzer/src/main/kotlin/managers/Bundler.kt:79]: commit-msg: For the case that Bundle**r**

File: analyzer/src/main/kotlin/managers/Bundler.kt [hunk 2]
```diff
@@ -146,11 +146,14 @@ class Bundler(
...
-                    arrayOf(
-                        "-S", "gem", "install", "--no-document", "--user-install", "$BUNDLER_GEM_NAME:$bundlerVersion"
-                    )
-                )
+                val output = runScriptCode("""
+                    require 'rubygems/commands/install_command'
+                    cmd = Gem::Commands::InstallCommand.new
+                    cmd.handle_options ["--no-document", "--user-install", "$BUNDLER_GEM_NAME:$bundlerVersion"]
+                    cmd.execute
+                """.trimIndent()).trim()
```
Comment (fviernau) [inline analyzer/src/main/kotlin/managers/Bundler.kt:154]: Commit-msg: Please provide a rationale "why" this is done.

**Review by fviernau — CHANGES_REQUESTED** [review]

File: analyzer/src/main/kotlin/managers/Bundler.kt (hunk 2 above)
Comment (sschuberth) [reply analyzer/src/main/kotlin/managers/Bundler.kt]: I've reworded the commit message a bit to hopefully make the rationale clearer.

**Review by fviernau — APPROVED** [review]

**Comment by sschuberth** [comment]
Merging despite the unrelated test failures.

**Comment by mnonnenmacher** [comment]
@sschuberth Package managers (and scanners) that support configuration options should always document them in the class docs, can you make a fixup for this? See for example `Pip`, so far the syntax for the option descriptions is consistent across all classes.

**Pull request merged**